scripts/functions/requirements/osx_brew

```diff
@@ -313,27 +313,31 @@ requirements_osx_brew_define_openssl()
   # OpenSSL version depends on the version of Ruby being installed
   undesired_check openssl098
 
-  case "$1" in
-    (ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*|ruby-1.8*)
-      brew_openssl_package="openssl"
-      ;;
-
-    (ree-1.8*)
-      brew_openssl_package="openssl"
-      ;;
-
-    (rbx-1*|rbx-2*)
-      brew_openssl_package="openssl"
-      ;;
-
-    (truffleruby*)
-      brew_openssl_package="openssl"
-      ;;
-
-    (*)
-      brew_openssl_package="openssl@1.1"
-      ;;
-  esac
+case "\$1" in
+  ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*|ruby-1.8*)
+    brew_openssl_package="openssl@1.1"
+    ;;
+
+  ree-1.8*)
+    brew_openssl_package="openssl@1.1"
+    ;;
+
+  rbx-1*|rbx-2*)
+    brew_openssl_package="openssl@1.1"
+    ;;
+
+  truffleruby*)
+    brew_openssl_package="openssl@1.1"
+    ;;
+
+  ruby-3.0*|ruby-3.1*)
+    brew_openssl_package="openssl@1.1"
+    ;;
```

Comment:
both ruby 1 and 2 are covered in the first switch, to be consistent might make sense to also cover 3 there as well?